### PR TITLE
Service documentation

### PR DESCRIPTION
**Pull request by obadrawi**
Adding console documentation for bundles

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,11 +7,11 @@ The SAP CP Extension Factory (XF) is designed to easily extend and mash up diffe
 ## Installation
 This addons repository as defined by [index.yaml](bundles/index.yaml) is installed to XF by default.
 
-If you need to install it manually, you need to upload the addon artifacts to a web server with location UPLOAD_DIR using `make preapre && make upload UPLOAD_DIR`. 
+If you need to install it manually, you need to upload the addon artifacts to a web server with location https://storage.googleapis.com/faros-prod-base-xf-bundles/index.yaml using `make preapre && make upload https://storage.googleapis.com/faros-prod-base-xf-bundles/index.yaml`.
```
Comment (a-thaler) [inline README.md]: Please revert that, makes no sense as the upload_dir is different then the download URL.

File: README.md [hunk 2]
```diff
@@ -23,6 +23,15 @@ To remove the addons from your Service Catalog:
 If you update the addons and re-upload them to your web server, you need to trigger a scan by the service catalog at your XF or Kyma cluster by calling:
 `svcat sync broker helm-broker --scope cluster`
 where svcat is the [Service Catalog CLI](https://svc-cat.io/docs/cli/)
+### Development Phases
+
+#### Pushing to Branch
+If you push to a branch then the addon artifactes are uploaded to the staging location only (https://storage.googleapis.com/faros-stage-base-xf-bundles/index.yaml).
+#### Merging to master
+If you merge a branch to the master then the addon artifactes are uploaded to the staging (https://storage.googleapis.com/faros-stage-base-xf-bundles/index.yaml) and pre-production (https://storage.googleapis.com/faros-int-base-xf-bundles/index.yaml) locations.
```
Comment (a-thaler) [inline README.md]: Only one of them, or?

File: bundles/commerce-mock-0.1.0/docs/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+docs:
+  - template:
+      displayName: "commerce-mock-0.1.0"
+      description: "Commerce Mock documentation"
```
Comment (a-thaler) [inline bundles/commerce-mock-0.1.0/docs/meta.yaml]: SAP Commerce Cloud Mock

File: bundles/commerce-mock-0.1.0/docs/overview.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+---
+title: Commerce Mock
```
Comment (a-thaler) [inline bundles/commerce-mock-0.1.0/docs/overview.md]: SAP Commerce Cloud Mock

File: README.md (hunk 2 above)
Comment (obadrawi) [reply README.md]: Just in case someone pushes to master directly?

File: README.md (hunk 1 above)
Comment (obadrawi) [reply README.md]: rverted

File: bundles/commerce-mock-0.1.0/docs/meta.yaml (hunk 3 above)
Comment (obadrawi) [reply bundles/commerce-mock-0.1.0/docs/meta.yaml]: done

File: bundles/commerce-mock-0.1.0/docs/overview.md (hunk 4 above)
Comment (obadrawi) [reply bundles/commerce-mock-0.1.0/docs/overview.md]: done

File: README.md [hunk 5]
```diff
@@ -11,7 +11,7 @@ If you need to install it manually, you need to upload the addon artifacts to a
 In your XF or Kyma cluster you then need to create a configMap using following commands:
 
 ```
-kubectl create configmap xf-addons-repo -n kyma-system --from-literal=URLs=https://UPLOAD_DIR/index.yaml
+kubectl create configmap xf-addons-repo -n kyma-system --from-literal=URLs=https://https://storage.googleapis.com/faros-prod-base-xf-bundles/index.yaml/index.yaml
```
Comment (a-thaler) [inline README.md]: There is twice https://

File: README.md (hunk 2 above)
Comment (a-thaler) [reply README.md]: but that is not implmented.. when I see the Jenkinsfile there will be one upload only

File: bundles/c4c-mock-0.1.0/docs/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,9 @@
+docs:
+  - template:
+      displayName: "c4c-mock-0.1.0"
+      description: "SAP C4C Mock documentation"
```
Comment (a-thaler) [inline bundles/c4c-mock-0.1.0/docs/meta.yaml]: Please see the root meta.yaml  for the correct name, something with Cloud for Customers

File: bundles/c4c-mock-0.1.0/docs/overview.md [hunk 7]
```diff
@@ -0,0 +1,6 @@
+---
+title: SAP C4C Mock
```
Comment (a-thaler) [inline bundles/c4c-mock-0.1.0/docs/overview.md]: Shouldn't the title be something like "Overview"

File: bundles/c4c-mock-0.1.0/docs/overview.md [hunk 8]
```diff
@@ -0,0 +1,6 @@
...
+type: Details
+---
+Use the c4c-mock application to mock an **SAP C4C** installation. The application is based on the [**Varkes**](https://github.com/kyma-incubator/varkes) project and is part of the [**XF-Application-Mocks**](https://github.com/SAP/xf-application-mocks).
```
Comment (a-thaler) [inline bundles/c4c-mock-0.1.0/docs/overview.md]: c4c-mock application -> SAP Cloud for ...

File: bundles/c4c-mock-0.1.0/docs/overview.md [hunk 9]
```diff
@@ -0,0 +1,6 @@
...
+After creating service instance, a new navigation node appears in the Kyma Console. Use the functionality it provides to connect the Mock to your Kyma cluster and browse the APIs.
```
Comment (a-thaler) [inline bundles/c4c-mock-0.1.0/docs/overview.md]: As the console integration is not done yet, please write how to access the UI: https://c4c-<namespace>.<cluster_domain>

File: Jenkinsfile [hunk 10]
```diff
@@ -38,10 +38,17 @@ pipeline {
...
             steps{
                 script{
+                    if(env.TAG_NAME)
+                    {
+                        upload('prod')
+                    }
                     if(env.BRANCH_NAME=="master"){
                          upload('int')
+                         upload('stage')   
+                    }
```
Comment (obadrawi) [inline Jenkinsfile]: @a-thaler it has both here?

File: bundles/c4c-mock-0.1.0/docs/overview.md (hunk 7 above)
Comment (obadrawi) [reply bundles/c4c-mock-0.1.0/docs/overview.md]: no it's the type that your talking about

File: README.md (hunk 5 above)
Comment (obadrawi) [reply README.md]: reverted to UPLOAD_DIR

File: Jenkinsfile (hunk 10 above)
Comment (a-thaler) [reply Jenkinsfile]: oh, right

File: bundles/c4c-mock-0.1.0/docs/overview.md (hunk 7 above)
Comment (a-thaler) [reply bundles/c4c-mock-0.1.0/docs/overview.md]: So where the title gets displayed?

**Pull request merged**